src/neovim/cursor.ts

```diff
@@ -117,7 +117,7 @@ export default class NeovimCursor {
         this.store.on('mode', () => this.updateCursorBlinking(this.store.mode !== 'insert'));
     }
 
-    updateSize() {
+    updateFontSize() {
```

Comment:
This method does not update font size. So IMO this method should be named as `onFontSizeUpdated` or `updateCursorSize`. What do you think?